admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/impl/SyncAuthTokenServiceImpl.java

```diff
@@ -264,7 +264,7 @@ private void validateRequestTimestamp(LocalDateTime requestTimestamp) {
                 break;
         }
 
-        ResponseWrapper<TokenResponseDto> responseWrapper = objectMapper.readValue(responseEntity.getBody(),
+        ResponseWrapper<TokenResponseDto> responseWrapper = objectMapper.readValue(null!=responseEntity?responseEntity.getBody():"",
```

Comment:
Not required to handle null check here, this is excepted to be handled by global exceptionHandler

Reply:
Done